Tests/TLSTests/LiveTests.swift

```diff
@@ -232,9 +206,8 @@ class LiveTests: XCTestCase {
             verifyHost: false,
             verifyCertificates: false
         )
-        
-        group.enter()
-        DispatchQueue.global(qos: .default).async {
+
+        background {
```

Comment:
we should make this accept throwing functions somehow if possible, it's annoying to work with sometimes right now, I have an idea for how to do it when I get some time